main/autobot/project_tracker.js

```diff
@@ -0,0 +1,37 @@
+var Handler = require('../autobot/handler');
+var JiraApi = require('jira').JiraApi;
+
+var JiraTracker = function (config) {
+    return new JiraApi('https',
+        config.host,
+        config.port,
+        config.user,
+        config.password,
+        'latest');
+}
+
+var ProjectTracker = function (name) {
+    this.tracker = JiraTracker({
+        host: 'lumoslabs.atlassian.net',
+        port: null,
+        user: 'alan@lumoslabs.com',
+        password: ''
+    });
+}
+
+ProjectTracker.prototype = {
+    getStory: function (storyId, callback) {
+        var handler = new Handler(callback);
```

Comment:
Hmmmmm we should maybe think about a place for these callbacks and maybe how to group them. 
[The status callback](https://github.com/abMatGit/hackathon-2016/pull/6/files#diff-e83dd419bac6347c97cc0f70a0bd1e22R26) is one example of many that we'll end up using. For the moment though i think they can all just live here or something.